docs/resources/ip_set.md

```diff
@@ -8,6 +8,8 @@ description: A resource that can be used to configure an IP set in NSX.
 
 This resources provides a way to configure an IP set in NSX. An IP set is a collection of IP addresses. It is often used in the configuration of the NSX firewall.
 
+This resource is applicable to NSX Policy Manager and is supported with NSX 9.0.0 onwards.
```

Comment:
This has been actually removed from the provider starting with 9.0!
We probably need a different PR for these resources, to state that they are NOT available from 9.0 onwards.
I can work on it.